packages/teams-js/test/public/video.spec.ts

```diff
@@ -23,38 +23,413 @@ describe('video', () => {
       app._uninitialize();
     }
   });
+  describe('registerForVideoFrame', () => {
+    let emptyVideoEffectCallback = (
+      _frame: video.VideoFrame,
+      _notifyVideoFrameProcessed: () => void,
+      _notifyError: (errorMessage: string) => void,
+    ): void => {};
+    const videoFrameConfig: video.VideoFrameConfig = {
+      format: video.VideoFrameFormat.NV12,
+    };
+
+    const allowedContexts = [FrameContexts.sidePanel];
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        return;
+      }
+      it('DESKTOP - should not allow registerForVideoFrame calls from the wrong context', async () => {
```

Comment:
would it be unreasonable to have different describe blocks for mobile and desktop within the registerforVideoFrame describe block? Just asking opinions

Reply:
This seems like a good idea to me

Reply:
Hey @Jessica-Koch, Is it okay to close the comment? We can talk further if you still have questions